mod_reforged/hooks/skills/backgrounds/character_background.nut

```diff
@@ -14,14 +22,25 @@
 		}];
 	}
 
+	// Overwrite to increase overall performance. The Wage for this background is no longer calculated newly during every update loop
+	q.onUpdate = @() function( _properties )
+	{
+		_properties.DailyWage = this.calculateDailyCost();
```

Comment:
You are still calculating the daily cost here, the same as vanilla, on every update loop. The only performance upgrade I see is that you no longer check for `trait.player` skill, which isn't enough of a performance upgrade to justify this overwrite of the entire function imo.

Reply:
My bad. The calculation was not supposed to happen here, I will fix that